ios/ZulipMobile.xcodeproj/project.pbxproj

```diff
@@ -923,110 +176,22 @@
-			);
-			name = Products;
-			sourceTree = "<group>";
-		};
-		A148FEDC1E9D8C9600479280 /* Products */ = {
-			isa = PBXGroup;
-			children = (
-				A148FEF91E9D8C9600479280 /* libRNSound.a */,
-			);
-			name = Products;
-			sourceTree = "<group>";
-		};
-		A14EA8551EACE4CE009D9E83 /* Products */ = {
-			isa = PBXGroup;
-			children = (
-				A14EA85B1EACE4CF009D9E83 /* libRCTAnimation.a */,
-				A14EA85D1EACE4CF009D9E83 /* libRCTAnimation.a */,
-			);
-			name = Products;
-			sourceTree = "<group>";
-		};
-		A155BFE81DD8E54100A8B695 /* Products */ = {
-			isa = PBXGroup;
-			children = (
-				A155BFEC1DD8E54100A8B695 /* libRCTCameraRoll.a */,
-			);
-			name = Products;
-			sourceTree = "<group>";
-		};
-		A1E053B11FDB490D002A87B7 /* Products */ = {
-			isa = PBXGroup;
-			children = (
-				A1E053B61FDB490E002A87B7 /* libART.a */,
-				A1E053B81FDB490E002A87B7 /* libART-tvOS.a */,
-			);
-			name = Products;
-			sourceTree = "<group>";
-		};
-		A1F9F7411E1BE26300A70FA5 /* Products */ = {
-			isa = PBXGroup;
-			children = (
-				A1F9F7571E1BE26300A70FA5 /* libRNDeviceInfo.a */,
-				0A955E261FACB50000801C8D /* libRNDeviceInfo-tvOS.a */,
-			);
-			name = Products;
-			sourceTree = "<group>";
-		};
 		A1FC272D1F75F05200546415 /* Recovered References */ = {
 			isa = PBXGroup;
 			children = (
-				3074EDA2D113487A925E187C /* libRNVectorIcons.a */,
-				D5664A74FA8048439CBAB734 /* libRNDeviceInfo.a */,
-				6DE2C81628724C8AA4862247 /* libRNSound.a */,
-				E518466F398E458DA2C685AF /* libSafariViewManager.a */,
-				C2F7E19951E64FC7B6BBE612 /* libRNFetchBlob.a */,
-				B83DDAD2507A4F0EB47660BD /* libRNSentry.a */,
-				09AC2386C1724D5BA93B51E9 /* libRNPhotoView.a */,
-				1AED53B5E6AA4E0AA52DB0C1 /* libRNSafeArea.a */,
-				5597847AA2A04FEE894C9C9E /* libRCTOrientation.a */,
-				875BCBF7DC0645ACBA61563B /* libRNImagePicker.a */,
-				BB41D085CB4A4C4580BA1877 /* libRNCWebView.a */,
-				C787C43F73EB496F95BBAD9A /* libRNCNetInfo.a */,
-				03CBDFCF298745C49268F40D /* libRNCAsyncStorage.a */,
-				AFC751239FBB487099FD7700 /* libLRDRCTSimpleToast.a */,
 			);
 			name = "Recovered References";
 			sourceTree = "<group>";
 		};
-		CFA67CF71EC232E70070048E /* Products */ = {
+		F8C26711A6226C32E43092A9 /* Pods */ = {
 			isa = PBXGroup;
 			children = (
-				CFA67D1B1EC232E70070048E /* libSafariViewManager.a */,
+				FB0511DD408657240F6D4EA3 /* Pods-ZulipMobile.debug.xcconfig */,
+				2C68A2E34977F3A580059AC7 /* Pods-ZulipMobile.release.xcconfig */,
+				047663CF32A867699B13342E /* Pods-ZulipMobileTests.debug.xcconfig */,
+				412BDA8A3FA264AE52430606 /* Pods-ZulipMobileTests.release.xcconfig */,
 			);
-			name = Products;
-			sourceTree = "<group>";
-		};
-		CFCFE50D1F0011FA00C295CF /* Products */ = {
-			isa = PBXGroup;
-			children = (
-				CFCFE52F1F0011FA00C295CF /* libRNFetchBlob.a */,
-			);
-			name = Products;
+			path = Pods;
 			sourceTree = "<group>";
 		};
 		F9E946BE712842C5A02C7F51 /* Resources */ = {
```

Comment:
This list retains references to the font files from react-native-vector-icons.

I think this particular list doesn't *do* anything -- I think it just controls the "Resources" expandable folder-like thing in the project view in the left sidebar. Not really sure, though.

Looks like these references can be deleted from that same left-sidebar UI, too.